bin/worker.php

```diff
@@ -41,14 +41,14 @@
 
 Addon::loadHooks();
 
-$spawn = (($_SERVER["argc"] == 2) && ($_SERVER["argv"][1] == "spawn"));
+$spawn = Argument::get('spawn',$_SERVER["argc"], false, 'boolean');
```

Comment:
`$_SERVER["argc"]` holds the argument count, not the argument array, please use `$_SERVER['argv']` instead.
Additionally, space is missing after a comma.